### DISCUSSION TITLE
Multimodule gradle project and with client generation.

### DISCUSSION THREAD

**Discussion by IvanDanilau** [Netflix/dgs-framework#922] [Q&A]
I have multimodule gradle projects. First, "Main" module has it's own graphql schema. which I set in configurations. Second module "Dependency" it's just a schema of another module, which I'm using for client generation (according to the recomendation in client generation docs).
But the problem is that when I launch "Main" and call graphiql , I see the queries of "Dependency" module. Even if I exclude "dependency" schemas from resources. And of course, don't put them in schema paths configuration. How I can avoid it?

**Comment by srinivasankavitha** [comment]
The client generation is part of code generation and independent. I assume you are referrring to schemaPaths that are set as part of the `generateJava` gradle task, correct? The recommendation will allow you to generate code for schema in different modules independently. The schema that is part of your service's GraphQL API is not affected by this configuration. The DGS will include all the schemas across all modules that are under the resources/schema directory.

Hope that clarifies the set up.
